niche_vlaanderen/niche.py

```diff
@@ -9,6 +9,7 @@
 import yaml
 import datetime
 import sys
+from pathlib import Path
 
 import numpy as np
 import numpy.ma as ma
```

Comment:
still required? Should we run flake8 to check for unused imports?

Reply:
pathlib is actually part or `write` method. Adding checks might be a good idea, but out of scope in this refactoring.

Reply:
Yes, I actually kept the usage in the plot-environment as it is convenient and will not influence the calculation speed.